clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -264,6 +226,10 @@ public EhcacheEntityResponse invoke(ClientDescriptor clientDescriptor, EhcacheEn
                                                   " Check your server configuration and define at least one offheap resource.");
       }
 
+      if (this.ehcacheStateService == null) {
+        throw new ServerMisconfigurationException("Server failed to retrieve EhcacheStateService .");
```

Comment:
Let's throw an `AssertionError` in the constructor for this - if it happens, it is serious.